sylvia-derive/src/entry_points.rs

```diff
@@ -42,13 +47,15 @@ impl<'a> EntryPointInput<'a> {
         Self { item, args }
     }
 
+    /// Process the input and generate the interface code.
```

Comment:
We could enrich this description with information about exporting several functions related to entrypoints